lib/iris/tests/__init__.py

```diff
@@ -91,6 +92,13 @@
     GRIB_AVAILABLE = True
 
 
+#: CDL history attribute regular expression pattern for ISO8601 timestamp.
+iso8601 = '(?P<lhs>\s*:history = ")' \
+    '(?P<timestamp>\d{4}\-?\d{2}\-?\d{2}' \
+    '(T\d{2}\:?\d{2}\:?\d{2}(\.\d{3})?(([\+\-]\d{2}\:?\d{2})|Z)?)?: )' \
+    '(?P<rhs>.*)'
```

Comment:
Indeed, I just noticed this below... still, if we could do so it would mean we could lose that (admittedly very impressive) but extremely cumbersome regex string!